backend/repository/data.js

```diff
@@ -329,3 +322,59 @@ exports.competenceMapping = async ({ dataplattformClient }) => {
     },
   };
 };
+
+exports.competenceCategories = async ({ dataplattformClient }) => {
+  // Request categories and competence values
+  const [reqCategories, reqCompetence] = await Promise.all([
+    dataplattformClient.report({
+      reportName: 'categories',
+    }),
+    dataplattformClient.report({
+      reportName: 'competenceAverage',
+    })
+  ]);
+  const [categoriesData, competenceData] = await Promise.all([
+    reqCategories.json(),
+    reqCompetence.json(),
+  ])
+
+  // Categories structure
+  let categories = {
+    "kategori" : "kompetansekartlegging",
+    "children" : []
+  }
+
+  // Get the main categories
+  const mainCategories = new Set(
+    categoriesData.flatMap(
+      item => Object.keys(item)
+    )
+  )
+
+  mainCategories.forEach(name => {
+    var categoryObject = {
+      "kategori": name,
+      "children": []
+    }
+    
+    // Get child categories
+    categoriesData.forEach(
+      item => {
+        const childName = item[name]
+        if (childName) {
+          // Create child category and merge competence data
+          const competance = competenceData[0][childName.toLowerCase()] || null
```

Comment:
jeg tror det heter competence istedenfor competance :) 

```suggestion
          const competence = competenceData[0][childName.toLowerCase()] || null
```